tests/python/unittest/test_operator.py

```diff
@@ -171,8 +171,29 @@ def test_regression():
                      lambda x: x,
                      lambda x, y : x - y)
 
+def test_swapaxes():
+	  print 'test swapaxes...'
+	  data = mx.symbol.Variable('data')
+	  shape = (2, 3, 4)
+	  data_tmp = np.ones(shape)
+	  data_tmp[0] = 1
+	  data_tmp[1] = 2
+	  arr_data = mx.nd.array(data_tmp)
+	  swap0 = mx.symbol.SwapAxis(data=data, dim1=0, dim2=2)
+	  swap = mx.symbol.SwapAxis(data=swap0, dim1=1, dim2=2)
+	  exe_c = swap.bind(mx.cpu(), args=[arr_data])
+	  exe_c.forward()
+	  out = exe_c.outputs[0].asnumpy()
+	  print data_tmp.shape
+	  print data_tmp
```

Comment:
Do assert, and compare with numpy result, testcases should be able to fail when we run it and something wrong happens. We cannot rely on eyebow to detect errors.

Something like

```
assert same(out, np.swapaxes(data_tmp, dim1, dim2))
```